src/game/hud.cpp

```diff
@@ -1308,12 +1312,34 @@ namespace hud
 
     void drawpointer(int w, int h, int index)
     {
+        bool use_custom_crosshair_colors = (crosshairweapons & 2) == 0;
```

Comment:
why "& 2" ?

Reply:
if (crosshairweapons & 2) == 0 then you didn't select an option that uses weapon colors. It was like that before, I just copied it